tools/generators/migrate-converged-pkg/schema.json

```diff
@@ -20,6 +20,11 @@
     "all": {
       "type": "boolean",
       "description": "Run generator on all vNext packages"
+    },
+    "owner": {
+      "type": "string",
+      "description": "Add particular team to CODEOWNERS file",
+      "pattern": "^@[/-\\w]+$"
```

Comment:
should this be `^@[\--\\w]+$` so that it includes the - in the team names? 
e.g. @microsoft/cxe-prg won't match because - is ascii 45 and / is ascii 47 so it's not in the range

Reply:
- you don't have to escape special characters in capture groups, the need of double escape `\w` is because it's in JSON
- all valid names will match (I verified) ![image](https://user-images.githubusercontent.com/1223799/159757873-3e8e94ea-f0c1-4b1c-9679-1a4071984719.png)